### ISSUE TITLE
Bunch of exceptions in tree chop down

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#19697]
![image](https://user-images.githubusercontent.com/3536496/101822723-a4fb1080-3ade-11eb-9a56-a10f356a082b.png)

**Comment by MazonDel** [comment]
@WeaselDog @Quadbro 

Per a discussion with @ptitducanh, the TreeFall.cs component was apparently added back to the TreeFallSfx prefab because they hadn't finished migrating all the falling tree audio functionality to the Wwise system. But with all the audio-clips removed from the project anyway, the component just fails since there is no way to give it the AudioClips it needs.

I removed this component and updated the prefab in Git, so the spammed error messages no longer exist. There's still a bit of dead-space in the audio, but there's a note concerning this in https://docs.google.com/spreadsheets/d/12HIHwauhOGyZ8lCrRrn4Q-Auy4XdVec3GwGn1t2AAIM/edit#gid=1417114153

It is possible that TreeFall.cs needs to be completely rewritten in order to manipulate Wwise for additional sounds, but maybe someone from the sound team knows had to nudge Wwise to doing what is required without needing that effort.

Possibly Related SideNote: I noticed a warning that "WoodDestroySfx doesn't support pooling. It may cause problems in runtime."
